infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/receive/LogReplicationMetadataManager.java

```diff
@@ -736,7 +738,9 @@ public void setReplicationStatusTable(LogReplicationSession session, long remain
                 SnapshotSyncInfo currentSnapshotSyncInfo;
                 if (snapshotStatus == null) {
                     log.warn("syncStatusPoller [snapshot] :: previous status is not present for session: {}", session);
-                    currentSnapshotSyncInfo = SnapshotSyncInfo.newBuilder().build();
+                    currentSnapshotSyncInfo = SnapshotSyncInfo.newBuilder()
```

Comment:
if we look at the definition of SnapshotSyncInfo in the protobuf, it says the following - 
"* If replication is in SNAPSHOT sync, this provides details of the
 * ongoing snapshot sync. If replication is in LOG ENTRY sync (delta).
 * this provides details of the previous SNAPSHOT sync leading up to
 * the log entry sync."

So whenever the syncType is Snapshot Sync, the previous snapshot sync info does not have any significance.  In both cases - here and on L744, currentSnapshotSyncInfo must be set in the same way